fed/mosaic-sumo/src/main/java/org/eclipse/mosaic/fed/sumo/bridge/facades/SimulationFacade.java

```diff
@@ -384,7 +384,13 @@ public TraciSimulationStepResult simulateUntil(long time) throws InternalFederat
                     vehicleData = new VehicleData.Builder(time, lastVehicleData.getName())
                             .copyFrom(lastVehicleData)
                             .position(veh.position.getGeographicPosition(), veh.position.getProjectedPosition())
-                            .stopped(vehicleStopMode).create();
+                            .stopped(vehicleStopMode)
+                            .route(veh.routeId)
+                            .movement(veh.speed, veh.acceleration, fixDistanceDriven(veh.distanceDriven, lastVehicleData))
+                            .orientation(DriveDirection.UNAVAILABLE, veh.heading, veh.slope)
+                            .consumptions(calculateConsumptions(veh, lastVehicleData))
+                            .emissions(calculateEmissions(veh, lastVehicleData))
```

Comment:
do we really need to compute consumption and emissions? could we just insert 0 consumption and 0 emissions? Or does SUMO actually compute something != 0?

Reply:
Yeah you are right, I just checked, both emissions and consumption are equal to 0 if the vehicle is parked